lib/faexport/views/docs.md

```diff
@@ -464,6 +464,9 @@ Retrieves information about the journal with the specified id.
 {
   "title": "Banner Update",
   "description": "Just a quick update... <snip>",
```

Comment:
This should be ` "description": "<div class=\"journal-body\">\n                                Just a quick update... <snip>",`

Reply:
Oh yeah, whoops, just pushed a fix to that